samples/snippets/upload_model_explain_image_managed_container_sample_test.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from uuid import uuid4
+
+import pytest
+
+import delete_model_sample
+
+import helpers
+
+import upload_model_explain_image_managed_container_sample
+
+PROJECT_ID = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+IMAGE_URI = "gcr.io/cloud-aiplatform/prediction/tf2-cpu.2-1:latest"
+ARTIFACT_URI = "gs://ucaip-samples-us-central1/model/cifar"
+DISPLAY_NAME = f"temp_upload_model_explain_image_managed_container_sample_{uuid4()}"
+
+INPUT_TENSOR_NAME = "bytes_inputs"
+OUTPUT_TENSOR_NAME = "output_0"
+
+
+@pytest.fixture
+def shared_state():
+    state = {}
+    yield state
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
replace the implementation of `teardown` with this: https://github.com/googleapis/python-aiplatform/blob/master/samples/snippets/upload_model_sample_test.py#L29-L31